ddl/ddl_api.go

```diff
@@ -1923,6 +1923,52 @@ func (d *ddl) RecoverTable(ctx sessionctx.Context, recoverInfo *RecoverInfo) (er
 	return errors.Trace(err)
 }
 
+// BuildTableInfoFromCreateViewAST builds model.TableInfo from a CreateViewStmt statement.
+func BuildTableInfoFromCreateViewAST(s *ast.CreateViewStmt) (*model.TableInfo, error) {
```

Comment:
How about renaming it to `BuildViewInfoFromAST` to correspond to `BuildTableInfoFromAST`?

Reply:
I don’t think this is a good name, because this function does not build `ViewInfo`, but `TableInfo`.